github/path_config_test.go

```diff
@@ -147,6 +147,41 @@ func testBackendPathConfigCreateUpdate(t *testing.T, op logical.Operation) {
 		assert.DeepEqual(t, testBaseURLValid, config.BaseURL)
 	})
 
+	t.Run("Exist Organization", func(t *testing.T) {
```

Comment:
You can use sub-tests for those two `Exist` tests, it seems only configuration differs, but not the logic.